.yarnrc

```diff
@@ -2,4 +2,4 @@
 # yarn lockfile v1
```

Comment:
This should be removed completely, shouldn't it?

Reply:
It will help people that still have `yarn v1` installed globally.